src/hooks/useAuth.tsx

```diff
@@ -7,18 +7,22 @@ import React, {
 } from 'react';
 import api, { getCSRF } from '../services/api';
 import { devPrint } from '../components/utils/RandomUtils';
+import { Member } from '../types';
+import { getCurrentUser } from '../services/member';
 
 interface AuthContextType {
-  isAuthenticated: boolean | null;
+  isAuthenticated: boolean;
```

Comment:
I don't really know why, but you can visit protected routes just fine, just not protected and dev routes.

I don't really like the idea of having `isAuthenticated` have a 3rd state, since you can really either be authenticated or not. Before authenticating, you are unauthenticated. I will fix `ProtectedRoute` to use member though

Reply:
Sure, but if the page loads before isAuthenticated changes to true it will redirect to login. That's why the 3rd state is unknown or undefined ;)

Reply:
> I will fix ProtectedRoute to use member though

check out ProtectedRoute